install/README.md

```diff
@@ -113,13 +56,12 @@
     kata-qemu   kata-qemu   6m7s
     ```
 
-* Login to the worker node and verify the status remote hypervisor service
+* View cloud-api-adaptor logs
 
     ```
-    sudo systemctl status remote-hyp.service
+    kubectl logs pod/cloud-api-adaptor-deployment-<aws|libvirt>-7c66d68484-zpnnw -n confidential-containers-system
```

Comment:
For the example, better to use the same pod name as displayed in the status above